### PR TITLE
chore(lint): add eslint to repo

### LINKED ISSUE #13: add eslint to this repo
Would it be a good idea to stabilize the styling for the experiments?

### PR DESCRIPTION
**Pull request by TDA**
Fixes #13
@vladikoff : Havent added Travis yet, cuz Peter said that could be a separate issue.
@pdehaan

### CODE REVIEW

**Comment by pdehaan** [comment]
Travis just puked all over the place, so, uh, lets add the .travis.yml as part of this PR.

```
Could not find .travis.yml, using standard configuration.
```

File: package.json [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {
-  "name":"fxa_content",
+  "name": "fxa_content",
   "version": "0.1.0",
   "private": true,
-  "license": "MPL 2.0"
+  "license": "MPL 2.0",
```
Comment (pdehaan) [inline package.json]: nit: "MPL-2.0" is the valid SPDX string.

Comment (TDA) [inline package.json]: I didnt touch that :O :O

Comment (TDA) [inline package.json]: changed. adding travis now.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "fxa/client",
+  "rules": {
+      "handle-callback-err": 0,
+      "strict": [2, "global"]
+    }
```
Comment (pdehaan) [inline .eslintrc]: @TDA: your indenting, it seems unbalanced.

File: package.json (hunk 1 above)
Comment (pdehaan) [inline package.json]: Hey, I'm not blaming anybody for MPL strings or anything. I'm just saying that npm@3 will start complaining, or something. ¯\_(ツ)_/¯

**Comment by pdehaan** [comment]
@TDA: Overall, looks good. I think your .travis.yml will probably just be something simple like this:

``` yaml
language: node_js

sudo: false

node_js:
  - "0.10"

script:
  - npm run lint
```

File: .eslintrc (hunk 2 above)
Comment (TDA) [inline .eslintrc]: :O :O :O I blame sublime! Trust me, I didnt do itttt!!!

**Comment by TDA** [comment]
@pdehaan I guess you were right, i simply c/p your travis file _sheepish grin_ MERGE!!@#!%#@$%

**Comment by pdehaan** [comment]
lgtm.
r? @vladikoff @dannycoates -- I have no merge authority in this repo.

File: package.json (hunk 1 above)
Comment (TDA) [inline package.json]: (y)

**Comment by vladikoff** [comment]
Excellent, thanks!

**Pull request merged**